Alert/Alert.js

```diff
@@ -140,12 +156,13 @@ const AlertBase = kind({
 	},
 
 	styles: {
-		css,
-		className: 'alert'
+		css: componentCss,
+		className: 'alert',
+		publicClassName: ['alert', 'content', 'fullscreen', 'title']
```

Comment:
It needs to be fixed to publicClassNames. When making multiple PRs for the same fix, please test them all on each branch.
```suggestion
		publicClassNames: ['alert', 'content', 'fullscreen', 'title']
```

Reply:
@vJIYEv I pushed commit to fix this typo.

Reply:
ok